azure-kusto-data/tests/test_helpers.py

```diff
@@ -0,0 +1,265 @@
+    true,
+    5678,
+    Infinity],
+    ["earliest pandas datetime",
+    "1677-09-21T00:12:43.145224193Z",
+    0,
+    true,
+    5678,
+    -Infinity],
+    ["latest pandas datetime",
+    "2262-04-11T23:47:16.854775807Z",
+    0,
+    true,
+    5678,
+    3.14159],
+    ["timedelta ticks",
+    "2021-12-22T11:43:00Z",
+    600000000,
+    true,
+    5678,
+    3.14159],
+    ["timedelta string",
+    "2021-12-22T11:43:00Z",
+    "1.01:01:01.0",
+    true,
+    5678,
+    3.14159],
+    [null,
+    "",
+    0,
+    false,
+    0,
+    0]]
+},
+{
+    "FrameType": "DataTable",
+    "TableId": 2,
+    "TableName": "QueryCompletionInformation",
+    "TableKind": "QueryCompletionInformation",
+    "Columns": [{
+        "ColumnName": "Timestamp",
+        "ColumnType": "datetime"
+    },
+    {
+        "ColumnName": "ClientRequestId",
+        "ColumnType": "string"
+    },
+    {
+        "ColumnName": "ActivityId",
+        "ColumnType": "guid"
+    },
+    {
+        "ColumnName": "SubActivityId",
+        "ColumnType": "guid"
+    },
+    {
+        "ColumnName": "ParentActivityId",
+        "ColumnType": "guid"
+    },
+    {
+        "ColumnName": "Level",
+        "ColumnType": "int"
+    },
+    {
+        "ColumnName": "LevelName",
+        "ColumnType": "string"
+    },
+    {
+        "ColumnName": "StatusCode",
+        "ColumnType": "int"
+    },
+    {
+        "ColumnName": "StatusCodeName",
+        "ColumnType": "string"
+    },
+    {
+        "ColumnName": "EventType",
+        "ColumnType": "int"
+    },
+    {
+        "ColumnName": "EventTypeName",
+        "ColumnType": "string"
+    },
+    {
+        "ColumnName": "Payload",
+        "ColumnType": "string"
+    }],
+    "Rows": [["2018-05-01T09:32:38.916566Z",
+    "unspecified;e8e72755-786b-4bdc-835d-ea49d63d09fd",
+    "5935a050-e466-48a0-991d-0ec26bd61c7e",
+    "8182b177-7a80-4158-aca8-ff4fd8e7d3f8",
+    "6f3c1072-2739-461c-8aa7-3cfc8ff528a8",
+    4,
+    "Info",
+    0,
+    "S_OK (0)",
+    4,
+    "QueryInfo",
+    "{\\"Count\\":1,\\"Text\\":\\"Querycompletedsuccessfully\\"}"],
+    ["2018-05-01T09:32:38.916566Z",
+    "unspecified;e8e72755-786b-4bdc-835d-ea49d63d09fd",
+    "5935a050-e466-48a0-991d-0ec26bd61c7e",
+    "8182b177-7a80-4158-aca8-ff4fd8e7d3f8",
+    "6f3c1072-2739-461c-8aa7-3cfc8ff528a8",
+    6,
+    "Stats",
+    0,
+    "S_OK (0)",
+    0,
+    "QueryResourceConsumption",
+    "{\\"ExecutionTime\\":0.0156222,\\"resource_usage\\":{\\"cache\\":{\\"memory\\":{\\"hits\\":13,\\"misses\\":0,\\"total\\":13},\\"disk\\":{\\"hits\\":0,\\"misses\\":0,\\"total\\":0}},\\"cpu\\":{\\"user\\":\\"00: 00: 00\\",\\"kernel\\":\\"00: 00: 00\\",\\"totalcpu\\":\\"00: 00: 00\\"},\\"memory\\":{\\"peak_per_node\\":16777312}},\\"dataset_statistics\\":[{\\"table_row_count\\":3,\\"table_size\\":191}]}"]]
+},
+{
+    "FrameType": "DataSetCompletion",
+    "HasErrors": false,
+    "Cancelled": false
+}]
+"""
+
+
+class TestDataFrameFromResultsTable(unittest.TestCase):
```

Comment:
We use pytest, so please remove the inheritance from unittest.TestCase, as it may interfere

Reply:
Done!